.github/actions/quic-interop-runner/action.yml

```diff
@@ -72,6 +72,15 @@ runs:
         # Don't fail CI if the interop test fails
```

Comment:
Do I understand correctly that long-term we would want to fail CI?

Reply:
Yes, once we get to the point that we consistently pass all tests (and all other stacks do to, when interop'ing with us).

Reply:
> (and all other stacks do to, when interop'ing with us).

We might want to pin their Docker images, such that a bug on their end doesn't break our CI. Not critical, thus I suggest keeping as is until this happens for the first time.